.gitattributes

```diff
@@ -0,0 +1,5 @@
+# go enforces lf line endings
+*.go eol=lf
+
+# testdata should not be mangled by git
+*.json binary
```

Comment:
Thanks for having a look at the code! I felt that since mangling of any kind is bad for file integrity, I tended towards marking them as binary.
We do expect all test files in the repo to be using LF line endings so mangling shouldn't be an issue anyway.
Would `eol=lf` be preferable to `binary`?

Reply:
+1, @torin-carey: I think that we should treat the test data as binary for exactly that reason

Reply:
Yes, agreed. I think we should change python-tuf to treat the test data as binary too. Thanks both.